pxr/usdImaging/usdImaging/resolvedAttributeCache.h

```diff
@@ -286,22 +286,36 @@ class UsdImaging_ResolvedAttributeCache
     // non-time varying data, entries may exist in the cache with invalid
     // values. The version is used to determine validity.
     struct _Entry {
-        _Entry()
+        _Entry() noexcept
```

Comment:
Could you comment on the addition of the noexcept keyword here? I'm not sure I'm spotting what necessitates the change?

Reply:
Oh right I think it'll still compile without it and we don't necessarily need it here, it is to imply the move will always be a move and not a copy. Maybe just have it for move and not the regular constructor?